ci/glci/aws.py

```diff
@@ -138,7 +138,7 @@ def register_image(
         EnaSupport=True,
         Name=image_name,
         RootDeviceName=root_device_name,
-        VirtualizationType='hvm', # | paravirtual
+        VirtualizationType='hvm' # | paravirtual
```

Comment:
This is a Python, not a JavaScript file. So trailing commas are valid.

Reply:
Regarding PEP: "Trailing commas are usually optional"